src/shared_modules/utils/threadDispatcher.h

```diff
@@ -88,13 +89,16 @@ namespace Utils
             {
                 if (m_running)
                 {
-                    m_queue.push
-                    (
-                        [value, this]()
+                    if (0 == m_maxQueueSize || m_queue.size() < m_maxQueueSize)
```

Comment:
In L92 it would not be:

 ```suggestion
                    if (0 != m_maxQueueSize && m_queue.size() < m_maxQueueSize)
```

If the maxQueueSize is 0 we sohouldn't push a new element on the queue, Am I right?